packages/eslint-plugin/src/rules/jsx-import.js

```diff
@@ -22,7 +22,8 @@ export default {
         sourceCode.ast.body.forEach(x => {
           if (
             x.type === 'ImportDeclaration' &&
-            x.source.value === '@emotion/core'
+            (x.source.value === '@emotion/react' ||
```

Comment:
do we need to include `@emotion/core` here? `pkg-renaming` rule should kick in first and then in a second pass this one could report a problem with `@emotion/react` import

Reply:
I think I saw cases where this rule was running before pkg-renaming and since this fixes it and there's no real harm to have it here we might as well leave it